Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/UserPreferences.java

```diff
@@ -142,6 +199,86 @@ public static void SavePrefsToIni(Context ctx)
+
+		String MainRadius_3 = prefs.getString("MainRadius_3", "100.000000");
+		String CRadius_3 = prefs.getString("CRadius_3", "100.000000");
+		String InputThres_3 = prefs.getString("InputThres_3", "90.000000");
+
+		// Current Wii analog range setup. Falls back to 50.000000 upon error.
+		String TiltRange_4 = prefs.getString("WiimoteTiltRange_4", "50.000000");
+		String TiltRange_5 = prefs.getString("WiimoteTiltRange_5", "50.000000");
+		String TiltRange_6 = prefs.getString("WiimoteTiltRange_6", "50.000000");
+		String TiltRange_7 = prefs.getString("WiimoteTiltRange_7", "50.000000");
+
+			// Current Nunchuk analog range setup. Falls back to 100.000000 upon error.
+		String NunchukRadius_4 = prefs.getString("NunchukRadius_4", "100.000000");
+		String NunchukRange_4 = prefs.getString("NunchukRange_4", "50.000000");
+
+		String NunchukRadius_5 = prefs.getString("NunchukRadius_5", "100.000000");
+		String NunchukRange_5 = prefs.getString("NunchukRange_5", "50.000000");
+
+		String NunchukRadius_6 = prefs.getString("NunchukRadius_6", "100.000000");
+		String NunchukRange_6 = prefs.getString("NunchukRange_6", "50.000000");
+
+		String NunchukRadius_7 = prefs.getString("NunchukRadius_7", "100.000000");
+		String NunchukRange_7 = prefs.getString("NunchukRange_7", "50.000000");
+
+			// Current Classic analog range setup. Falls back to 100.000000 (90 for triggers) upon error.
+		String ClassicLRadius_4 = prefs.getString("ClassicLRadius_4", "100.000000");
+		String ClassicRRadius_4 = prefs.getString("ClassicRRadius_4", "100.000000");
+		String ClassicThres_4 = prefs.getString("ClassicThres_4", "90.000000");
+
+		String ClassicLRadius_5 = prefs.getString("ClassicLRadius_5", "100.000000");
+		String ClassicRRadius_5 = prefs.getString("ClassicRRadius_5", "100.000000");
+		String ClassicThres_5 = prefs.getString("ClassicThres_5", "90.000000");
+
+		String ClassicLRadius_6 = prefs.getString("ClassicLRadius_6", "100.000000");
+		String ClassicRRadius_6 = prefs.getString("ClassicRRadius_6", "100.000000");
+		String ClassicThres_6 = prefs.getString("ClassicThres_6", "90.000000");
+
+		String ClassicLRadius_7 = prefs.getString("ClassicLRadius_7", "100.000000");
+		String ClassicRRadius_7 = prefs.getString("ClassicRRadius_7", "100.000000");
+		String ClassicThres_7 = prefs.getString("ClassicThres_7", "90.000000");
+
+			// Current Guitar analog range setup. Falls back to 100.000000 upon error.
+		String GuitarRange_4 = prefs.getString("GuitarRange_4", "100.000000");
+
+		String GuitarRange_5 = prefs.getString("GuitarRange_5", "100.000000");
+
+		String GuitarRange_6 = prefs.getString("GuitarRange_6", "100.000000");
+
+		String GuitarRange_7 = prefs.getString("GuitarRange_7", "100.000000");
+
+			// Current Drums modifier Radius setup. Falls back to 100.000000 upon error.
+		String DrumsRadius_4 = prefs.getString("DrumsRadius_4", "100.000000");
+
+		String DrumsRadius_5 = prefs.getString("DrumsRadius_5", "100.000000");
```

Comment:
There's no need for newlines between these. It's inconsistent with all other controls.